stdlib/LibCURL_jll/src/LibCURL_jll.jl

```diff
@@ -9,41 +9,56 @@ if !(Sys.iswindows() || Sys.isapple())
     using OpenSSL_jll
 end
 
-const PATH_list = String[]
-const LIBPATH_list = String[]
-
 export libcurl
 
 # These get calculated in __init__()
 const PATH = Ref("")
+const PATH_list = String[]
 const LIBPATH = Ref("")
+const LIBPATH_list = String[]
 artifact_dir::String = ""
-libcurl_handle::Ptr{Cvoid} = C_NULL
 libcurl_path::String = ""
 
+if Sys.isfreebsd()
+    _libcurl_dependencies = LazyLibrary[]
```

Comment:
All of these zero-dependency lists for FreeBSD are surprising - why is it so common for that platform?

Reply:
I don't know. They are all empty, actually. 
Also note that the new Zstd_jll also isn't a dep of libLLVM so FreeBSD is erroring.

@staticfloat I didn't see any obvious place the testing could be incorrect?

Reply:
Apparently there's a readelf styling difference on freebsd where it shows as `NEEDED` not `(NEEDED)`

Reply:
FreeBSD wants libcxxrt for several libs but I don't know what provides that?
```
From worker 9:	┌ Warning: Dependency mismatch
--
  | From worker 9:	│   jll = "GMP_jll"
  | From worker 9:	│   library = "libgmpxx"
  | From worker 9:	│   missing_deps = "libgmp, libc++, libcxxrt, libgcc_s"
  | From worker 9:	│   extraneous_deps = ""
  | From worker 9:	│   actual_deps = "libgmp, libc++, libcxxrt, libgcc_s"
  | From worker 9:	└ @ Main.Test75Main_stdlib_dependencies /usr/home/julia/.buildkite-agent/builds/freebsd13-amdci6-0/julialang/julia-master/julia-

```

Reply:
I see it came up here https://github.com/JuliaPackaging/Yggdrasil/pull/2108#issuecomment-731304853 but I don't see how it was resolved

Reply:
@giordano pointed out it's an OS lib https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/d10ad161d40260a8aa942b81c100e2bacc462ee9/src/auditor/dynamic_linkage.jl#L215